posthog/api/team.py

```diff
@@ -150,19 +148,14 @@ def create(self, validated_data: Dict[str, Any], **kwargs) -> Team:
             raise e
         return team
 
-    def _handle_timezone_update(self, team: Team, new_timezone: str) -> None:
-        hashes = (
-            Insight.objects.filter(team=team, last_refresh__gt=now() - relativedelta(days=7))
-            .exclude(filters_hash=None)
-            .values_list("filters_hash", flat=True)
-        )
+    def _handle_timezone_update(self, team: Team) -> None:
+        # :KLUDGE: This is incorrect as it doesn't wipe caches not currently linked to insights. Fix this some day!
```

Comment:
The Redis results have a 7 day TTL so we're pretty much OK to ignore them.

Should we be deleting the `InsightCachingStates` too?